IDEAS/Buildings/Components/OuterWall.mo

```diff
@@ -26,8 +28,16 @@ model OuterWall "Opaque building envelope construction"
   final parameter Real U_value=1/(1/8 + sum(constructionType.mats.R) + 1/25)
     "Wall U-value";
 
-  parameter Real coeffsCp[:,:]=[0,0.4; 45,0.1; 90,-0.3; 135,-0.35; 180,-0.2; 225,
-      -0.35; 270,-0.3; 315,0.1; 360,0.4]
+
+  replaceable parameter
+    IDEAS.Buildings.Data.WindPressureCoeff.Lowrise_Square_Exposed Cp_table
```

Comment:
Why a table instead of the previous solution?

Reply:
The previous solution was just a default coeffCp values, but only was a good default for vertical walls. For roofs, the coeffCp values needed to be changed manually which as such is maybe not the best default. This solution is proposed as improvement to the last one as this allows to change the coeffCp with surface tilt based on one general table.

It is still possible to override the Cp values on component level for the expert user but for someone who just wants to use the default,  1 table can be used for the whole building (common solution).